src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,41 @@
+import { ProfileInterface } from "@/interfaces/ProfileInterface";
+import {
+  ProfileContainer,
+  ProfileContent,
+  ProfileLinks,
+  ProfileDescription,
+  ProfileImage,
+  ProfileName,
+} from "./Profile.styles";
+
+const Profile = (data: ProfileInterface) => {
```

Comment:
```suggestion
const Profile = ({ data }: { data: ProfileInterface }) => {
```

Destructuring the data link this we're destructuring the given prop argument.